test_scripts/API/Restructuring_OnResetTimeout/017_BC_OnResetTimeout_GENERIC_ERROR_timeout_is_reset_for_rpc_with_customTimeout.lua

```diff
@@ -0,0 +1,49 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+--
+-- Description:
+-- In case:
+-- 1) RPC with own timeout is requested
+-- 2) SDL re-sends this request to HMI and wait for response
+-- 3) Default (10 sec) timeout is expired
+-- 4) HMI sends BC.OnResetTimeout(resetPeriod = 7000) to SDL
```

Comment:
if there is no step 4, what's the expected timeout of SDL response GENERIC_ERROR for the request? is it `default timeout + own timeout`?

and because of step7, the `own timeout` changed to 7 seconds?